src/java.desktop/share/classes/sun/java2d/cmm/CMSManager.java

```diff
@@ -100,33 +100,19 @@ public Profile loadProfile(byte[] data) {
             return p;
         }
 
-        public int getProfileSize(Profile p) {
```

Comment:
Not sure if it is a big loss, but it looks like we are no longer printing profile and tag sizes under CMMTracer after the fix.